tests/performance/test_footprint.py

```diff
@@ -1,13 +1,15 @@
 import platform
 import time
+from typing import Any, AsyncGenerator
 
 import aiomcache
 import pytest
 import redis.asyncio as redis
 
 
 @pytest.fixture
-async def redis_client() -> redis.Redis:
+async def redis_client() -> "AsyncGenerator[redis.Redis[Any], None]":
```

Comment:
```suggestion
async def redis_client() -> "AsyncIterator[redis.Redis[Any]]":
```

Reply:
Done (and then reverted per subsequent comment)